src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -509,6 +509,15 @@ static constexpr Property<uint32_t, PropertyMutability::RW> auto_batch_timeout{"
 static constexpr Property<std::tuple<unsigned int, unsigned int, unsigned int>, PropertyMutability::RO>
     range_for_async_infer_requests{"RANGE_FOR_ASYNC_INFER_REQUESTS"};
 
+/**
+ * @brief Read-write property to set whether force terminate tbb when ov core destruction
+ * value type: boolean
+ *   - True explicitly terminate tbb when ov core destruction
+ *   - False will not involve additional tbb operations when core destruction
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<bool, PropertyMutability::RW> force_tbb_terminate{"FORCE_TBB_TERMINATE"};
```

Comment:
Please add bindings to it here:
https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/pyopenvino/core/properties/properties.cpp

Without it Python API will lack new property. Tests for it could be useful as well.

Reply:
sure, will do it!

Reply:
done